debian/copyright

```diff
@@ -0,0 +1,50 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/freedombox/Plinth
+Upstream-Name: Plinth
+Files-Excluded: plinth/modules/jsxc/static/img plinth/modules/jsxc/static/libjs-jsxc/*
```

Comment:
Should the Files-Excluded line be removed?

I was using it to avoid lintian errors about unsafe links in source package. The links are re-created with debian/plinth.links.